### DISCUSSION TITLE
Hello everybody and welcome!

### DISCUSSION THREAD

**Discussion by ooples** [ooples/OoplesFinance.StockIndicators#12] [Announcements]
I'm launching this feature as a way to keep in touch with the community so this is the perfect place to ask questions about indicators you think are useful, feedback on various features in the project, or just introducing yourself to everyone else!

I'm Franklin and I have been a C# developer for many years. I love the world of finance and technical analysis especially. I publish scripts very often on tradingview.com as cheatcountry and I want to make this project a resource for any developer who wants to use any stock indicator from our vast shared collection. Be sure to introduce yourself as well!

**Comment by Crodomi** [comment]
Hey hello, Buddy I really like one of yours indicators, rate of change bands to be specific.
 I have zero experience in coding, tried chatgpt to edit and it was a disaster, could you add one more roc into the same script to work apart from the other one. Or the possibility to add lookback to the bands please
